docs/navigation-parameters-reference.md

```diff
@@ -1,294 +1,305 @@
+- **attributes**:
+  - **preloadUrl**(string): needs to be an absolute URL for a node from the view group. It is recommended that you use a dedicated small, visually empty view, which imports Luigi Client and is fine with getting an empty context, for example, without an access token. The **preloadUrl** property is also required for view group caching in case you need a view group iframe to refresh whenever you navigate back to it.
+
+
+## Node properties
+Node properties are all the properties that can be added to an individual navigation node in the `nodes:` section of the Luigi configuration file.
+
+### pathSegment
+- **type**: string
+- **description**: specifies the partial URL of the current segment. **pathSegment** must not contain slashes.
+- **examples**:
+  - A static pathSegment of value `settings` results in `yourwebsite.com/settings`.
+  - A dynamic pathSegment is prefixed with a colon and can load any value. Find out more about dynamic paths in Luigi [here](navigation-configuration.md#creating-a-dynamic-path).
+
+### link
+- **type**: string
+- **description**: refers to an absolute path in the navigation structure or a relative path to a grandchild of the current path. If this property is defined, **pathSegment** is ignored.
+
+### externalLink
+- **type**: object
+- **description**: indicates that the node links to an external URL. If this property is defined, **pathSegment** and **link** properties are ignored.
+- **attributes**:
+  - **sameWindow** defines if the external URL is opened in a new or current tab. The default value for this property is `false`.
+  - **url** is the external URL that the node leads to.
+
+### label
+- **type**: string
+- **description**: contains the display name of the navigation node.
+
+### testId
+- **type**: string
+- **description**: allows you define your own custom `testId`. If there is nothing specified, it is a combination of the node's pathSegment followed by a dash (if pathSegment exists) and the label written as one word and lower case (e.g. `pathsegment_label` or `label`).
+
+### hideFromNav
+- **type**: boolean
+- **description**: shows or hides a navigation node. You can still navigate to the node but it does not show up in the top or left pane.
+
+### viewUrl
+- **type**: string
+- **description**: contains the URL or path to a view which renders when you click the navigation node. Use either a full URL or a relative path. This value may consist of variables if you have specified a **navigationContext** with a dynamic **pathSegment**. If **viewUrl** is undefined, Luigi activates the child node specified in **defaultChildNode**. When both **viewUrl** and **defaultChildNode** are undefined, Luigi opens the first child of the current node.
```

Comment:
Hm, the viewUrl can contain variables from:
- dynamic path segment (with ':' prefix, e.g. :productId)
- node params, in form {nodeParams.paramName}
- context, in form {context.contextPropertyName}

It has nothing to do with the navigationContext.